docs/articles/documentation/extending-testcafe/custom-reporter/creating-reporter-plugin/helper-methods.md

```diff
@@ -0,0 +1,297 @@
+
+### newline
+
+Adds a new line to the report.
+
+```text
+newline () → this
+```
+
+**Example**
+
+```js
+reportTaskDone (endTime, passed) {
+    ...
+    var summary = passed === this.testCount ?
+                  `${this.testCount} passed` :
+                  `${this.testCount - passed}/${this.testCount} failed`;
+
+    this.newline()
+        .write(summary)
+        .newline();
+}
+```
+
+### write
+
+Writes the specified text in the report.
+
+```text
+write (text) → this
+```
+
+Parameter | Type   | Description
+--------- | ------ | ---------------------------------
+`text`    | String | The text to output in the report.
+
+**Example**
+
+```js
+reportTaskDone (endTime, passed) {
+    ...
+    var summary = passed === this.testCount ?
+                  `${this.testCount} passed` :
+                  `${this.testCount - passed}/${this.testCount} failed`;
+
+    this.write(summary)
+        .newline();
+}
+```
+
+### useWordWrap
+
+Toggles word wrapping for subsequent output.
+
+```text
+useWordWrap (use) → this
+```
+
+Parameter | Type    | Description
+--------- | ------- | ---------------------------------------
+`use`     | Boolean | Specify *true* to enable word wrapping.
+
+**Example**
+
+The following example demonstrates how to enable word wrapping in case when the `title` string might be long.
+
+```js
+reportTestDone (name, errs) {
+    ...
+    name = `${this.currentFixtureName} - ${name}`;
+
+    var title = `${result} ${name}`;
+
+    this.useWordWrap(true)
+        .write(title);
+}
+```
+
+### setIndent
+
+Specifies indentation for subsequent output.
+
+```text
+setIndent (val) → this
+```
+
+Parameter | Type    | Description
+--------- | ------- | ----------------------------------------------------
+`val`     | Integer | Specifies the number of spaces to indent a new line.
+          |         | To disable indentation, set the parameter to 0.
+
+**Example**
+
+The following example demonstrates how to indent a summary information with 4 spaces.
+
+```js
+reportTaskDone (endTime, passed) {
+    ...
+    var summary = passed === this.testCount ?
+                  `${this.testCount} passed` :
+                  `${this.testCount - passed}/${this.testCount} failed`;
+
+    this.setIndent(4)
+        .write(summary);
+}
+//=>    2/6 failed
+```
+
+## Coloring Helper
+
+### chalk
+
+[chalk](https://github.com/chalk/chalk) is used for ANSI-coloring of the text.
+
+```text
+chalk.<style>[.<style>...](string, [string...]) → this
+```
+
+**Example**
+
+The following example demonstrates how to color test results.
+
+```js
+reportTestDone (name, errs) {
+    var hasErr    = !!errs.length;
+    var result    = hasErr ? this.chalk.green(`passed`) : this.chalk.red(`failed`);
+
+    name = `${this.currentFixtureName} - ${name}`;
+
+    var title = `${result} ${name}`;
+
+    this.write(title);
+}
+```
+
+To disable `chalk` and any coloring in reports, set the reporter's `noColors` property in the `src/index.js` file to `false`.
```

Comment:
It's **force disable**, we need to describe that it's useful for the machine-readable formats. And it needs example. Because it's not quire clear how this property should be set